tests/main/security-seccomp/task.yaml

```diff
@@ -0,0 +1,64 @@
+summary: Check basic seccomp rules
+
+# This test verifies that:
+# - absence of a syscall is denied
+# - use of a bare syscall (ie, no arguments) is allowed
+# - use of a syscall with arg filtering is allowed with matching arguments
+# - use of a syscall with arg filtering is denied with unmatching arguments
+#
+# We choose the setpriority syscall for these tests since it is available on
+# all architectures and can be easily used to test all of the above. As part of
+# its testing, this test will modify the snap's seccomp filter and recompile
+# it. On strict mode systems, it will also adjust the apparmor profile to allow
+# 'capability nice' so it doesn't interfere with anything.
+
```

Comment:
```
2019-04-23 22:14:23 Failed tasks: 2
    - google:debian-sid-64:tests/main/security-seccomp
    - google:fedora-29-64:tests/main/security-seccomp
```
Looks like we need to disable:
```
systems: [-fedora-*, -debian-*]
```
or wait for fixes in respective distros to land.

Reply:
@bboozzoo - yes, that is a discussion point. This is demonstrating that the check is finding the issue. Now we need to decide what to do about it.

Reply:
On second thought, it's probably better to execute the test everywhere and make sure that we anticipate the broken behavior, so we'll know when the fixes trickle down to specific distros.